src/handlers/testing/align/components/AutogeneratedEvaluatorsList.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Text } from 'ink';
+import { type AutogeneratedEvaluator } from '../util/models';
+
+export function AutogeneratedEvaluatorsList(props: {
+  evaluators: AutogeneratedEvaluator[];
+}) {
+  return (
+    <Box flexDirection="column">
+      <Text>Generated {props.evaluators.length} evaluators:</Text>
```

Comment:
Do we need an empty state if there are 0 generated evaluators?

Reply:
i'd throw that in the error handling bucket, currently there is no error handling at all, like if there is an uncaught error in your code, in our API requests, etc. will probably tackle as a follow up